src/main/java/org/biojava/examples/SimpleExample.java

```diff
@@ -0,0 +1,39 @@
+package org.biojava.examples;
+
+import org.biojava.spark.data.AtomDataRDD;
+import org.biojava.spark.data.AtomSelectObject;
+import org.biojava.spark.data.SparkUtils;
+import org.biojava.spark.data.StructureDataRDD;
+
+/**
+ * A very simple example reading the PDB and printing the 
+ * composition of elements.
+ * @author Anthony Bradley
+ *
+ */
+public class SimpleExample {
+
+	public static void main(String[] args) {
+		
+		Long startTime = System.currentTimeMillis();
+		// Get the RDD of the atom contacts
+		AtomDataRDD atomDataRDD = new StructureDataRDD("/Users/anthony/full")
```

Comment:
I was thinking it would also be cool to provide a small (50mb) hadoop file as part of the JAR - so people can get going without  a download.